lmn/templates/lmn/notes/note_detail.html

```diff
@@ -12,4 +12,44 @@ <h2 id="note-page-title">{{ note.show.artist.name }} at {{ note.show.venue.name
       <p id="note-photo"><img src="{{ note.photo.url }}"></p>
 {% endif %}
 
+{% if note.rating == 1 %}
```

Comment:
Should also have the star presented even there is no rating being given yet or write some statement saying no rating